<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EasySaveUI</class>
 <widget class="QMainWindow" name="EasySaveUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="path_lineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="browse_pushButton">
        <property name="text">
         <string>Browse ...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>File list : </string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="back_pushButton">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="forward_pushButton">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="listWidget"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QCheckBox" name="filter_checkBox">
        <property name="text">
         <string>File format filter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="filter_lineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="customEnable_checkBox">
      <property name="text">
       <string>Use custom name</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="1">
       <widget class="QCheckBox" name="custom1_checkBox">
        <property name="text">
         <string>Custom1</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLineEdit" name="e1_lineEdit">
        <property name="text">
         <string>-3</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QCheckBox" name="e3_checkBox">
        <property name="text">
         <string>E3</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLineEdit" name="e3_lineEdit">
        <property name="text">
         <string>-1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="e2_lineEdit">
        <property name="text">
         <string>-2</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QCheckBox" name="e2_checkBox">
        <property name="text">
         <string>E2</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="e1_checkBox">
        <property name="text">
         <string>E1</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLineEdit" name="customText2_lineEdit"/>
      </item>
      <item row="1" column="3">
       <widget class="QCheckBox" name="version_checkBox">
        <property name="text">
         <string>Version</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="customText1_lineEdit">
        <property name="text">
         <string>custom</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QCheckBox" name="custom2_checkBox">
        <property name="text">
         <string>Custom2</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="version_lineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="name_lineEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Format: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="format_comboBox"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="open_pushButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="save_pushButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Save ++</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>436</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>e1_checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>e1_lineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>344</y>
    </hint>
    <hint type="destinationlabel">
     <x>66</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>e2_checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>e2_lineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>342</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>e3_checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>e3_lineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>378</x>
     <y>343</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>340</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>custom1_checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>customText1_lineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>369</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>365</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
